src/lib/controllers/ui-controller.js

```diff
@@ -390,6 +393,9 @@ export default class UIController {
     document.body.classList.add('alpheios')
 
     await Promise.all(optionLoadPromises)
+
+    this.registerAndActivateMouseMove('GetSelectedText', this.options.textQuerySelector)
```

Comment:
Placed it here - after settings are loaded - to be able tio check them before activation